lib/tasks/gettext.rake

```diff
@@ -30,7 +32,7 @@ begin
       end
 
       def files_to_translate
-        Dir.glob("#{@engine.root}/{app,db,lib,config,locale}/**/*.{rb,erb,haml,slim,rhtml,js}")
+        Dir.glob(File.join(@engine.root, FILE_GLOB))
```

Comment:
Doesn't this also work (since `root` is a Pathname):
```suggestion
        @engine.root.glob(FILE_GLOB)
```

Reply:
It does, thanks